lambdas/reset-identity/src/test/java/uk/gov/di/ipv/core/resetidentity/ResetIdentityHandlerTest.java

```diff
@@ -83,18 +80,61 @@ void setUpEach() {
     }
 
     @Test
-    void shouldDeleteUsersVcsAndReturnNext() throws SqsException {
+    void handleRequest_whenFraudReset_shouldDeleteUsersVcsAndReturnNext() throws SqsException {
```

Comment:
Not a big deal but it's not just fraud reset that calls this lambda so maybe just `whenNotUserInitiated` ?